bin/webpack.js

```diff
@@ -29,35 +29,57 @@ try {
 	webpackCliInstalled = false;
 }
 
+let webpackCommandInstalled = false;
+try {
+	require.resolve("webpack-command");
+	webpackCommandInstalled = true;
+} catch (err) {
+	webpackCommandInstalled = false;
+}
+
 if (!webpackCliInstalled) {
 	const path = require("path");
 	const fs = require("fs");
 	const readLine = require("readline");
+
+	console.error(
+		"The CLI moved into a separate packages.\nWe have two CLI packages:\n- webpack-cli (https://github.com/webpack/webpack-cli). The original webpack CLI moved from webpack@3.\n- webpack-command (https://github.com/webpack-contrib/webpack-command). The lightweight, modular, and opinionated webpack CLI."
```

Comment:
It might be clearer to users to say something like:

```
The CLI for webpack must be installed as a separate package, for which there are two choices:
    webpack-cli (https://github.com/webpack/webpack-cli) : The original webpack CLI from webpack@3
    webpack-command (https://github.com/webpack-contrib/webpack-command) : A lightweight, opinionated webpack CLI.
```

Reply:
@shellscape done :+1: